packages/liferay-npm-scripts/src/scripts/lint.js

```diff
@@ -14,11 +16,18 @@ const LINT_PATHS = getMergedConfig('npmscripts').lint;
  * @param {boolean} fix Specify if the linter should auto-fix the files
  */
 module.exports = function(fix) {
-	const args = [...LINT_PATHS];
+	const CONFIG_PATH = path.join(process.cwd(), 'TEMP-prettier-config.json');
 
-	if (fix) {
-		args.push('-i');
-	}
+	fs.writeFileSync(CONFIG_PATH, JSON.stringify(getMergedConfig('prettier')));
 
-	spawnSync('csf', args);
+	const args = [
+		'--config',
+		CONFIG_PATH,
+		fix ? '--write' : '--check',
+		...LINT_PATHS
+	];
+
+	spawnSync('prettier', args, {}, true);
+
+	fs.unlinkSync(CONFIG_PATH);
```

Comment:
We'll want a `try`/`finally` here so that we don't leave a turd 🤭 on the the filesystem when/if things go wrong.